tools/debian_buildenv.sh

```diff
@@ -27,6 +27,17 @@ case "$COMMAND" in
         esac
 
         sudo apt-get update
+
+	# If jackd2 is installed as per dpkg database, install libjack-jackd2-dev.
+        # This avoids a package deadlock, resulting in jackd2 being removed, and jackd1 being installed,
+        # to satisfy portaudio19-dev's need for a jackd dev package. In short, portaudio19-dev needs a
+        # jackd dev library, so let's give it one..
+        if [ "$(dpkg-query -W -f='${Status}' jackd2 2>/dev/null | grep -c "ok installed")" -eq 1 ]; 
```

Comment:
```suggestion
        if [ "$(dpkg-query -W -f='${Status}' jackd2 2>/dev/null | grep -c "ok installed")" -eq 1 ];
```
extra space at the end of the line